### ISSUE TITLE
Fails to flash from URL with Zip version not supported error

### ISSUE THREAD

**Issue by tve** [balena-io/etcher#3634]
- 1.6.0 Arch linux
- Arch linux x64
- https://motus-builds.s3.amazonaws.com/tve/sensorgnome-build/pimod/sg-armv7-rpi-buster-2021-320.zip

If I try to flash the above image from URL I get an error "Zip version 78.8 is not supported". But if I download the zip and flash from file it works fine, which leads me to believe the problem isn't in the zip file. What on earth is zip version 78.8? Is there a work-around for this?

Debug console:
```
Successfully connected to IPC server: etcher-server-209115, socket root /run/user/1000/
gui.js:89974 Image: https://motus-builds.s3.amazonaws.com/tve/sensorgnome-build/pimod/sg-armv7-rpi-buster-2021-320.zip
gui.js:89974 Devices: /dev/sdf
gui.js:89974 Auto blockmapping: true
gui.js:89974 Decompress first: true
gui.js:58569 Tue Nov 16 2021 14:30:58 GMT-0800 (Pacific Standard Time) Write failed ({"image":{"size":3342860288,"compressedSize":917713062,"hasMBR":true,"path":"https://motus-builds.s3.amazonaws.com/tve/sensorgnome-build/pimod/sg-armv7-rpi-buster-2021-320.zip","partitions":[{"offset":4194304,"size":268435456,"type":12,"index":1},{"offset":272629760,"size":3070230528,"type":131,"index":2}],"isCompressed":true,"name":"sg-armv7-rpi-buster-2021-320.img","extension":"zip"},"drives":[{"size":32010928128,"isVirtual":false,"enumerator":"lsblk:json","logicalBlockSize":512,"raw":"/dev/sdf","error":null,"isReadOnly":false,"displayName":"/dev/sdf","blockSize":512,"isSCSI":false,"isRemovable":true,"device":"/dev/sdf","busVersion":null,"isSystem":false,"busType":"USB","isCard":null,"isUSB":true,"devicePath":"/dev/disk/by-path/pci-0000:06:00.0-usb-0:1.4.3:1.0-scsi-0:0:0:0","mountpoints":[{"path":"/run/media/tve/boot","label":"boot"},{"path":"/run/media/tve/rootfs","label":"rootfs"},{"path":"/run/media/tve/DATA","label":"DATA"}],"description":"Kingston UHSII uSD Reader (boot, rootfs, DATA)","isUAS":null,"partitionTableType":"mbr"}],"driveCount":1,"uuid":"52e7c5e6-262d-45ee-9420-c614b47f2696","status":"failed","flashInstanceUuid":"52e7c5e6-262d-45ee-9420-c614b47f2696","error":{"name":"Error"},"sample":0.1,"applicationSessionUuid":"d7fe4156-f394-4201-aa8b-4a2d0f34e2dc","flashingWorkflowUuid":"995a8172-05e0-40b9-a603-3f959e1adcd9"})
gui.js:58639 {"stack":"Error: Zip version 78.8 is not supported\n    at Immediate.<anonymous> (/usr/lib/balena-etcher/generated/child-writer.js:24476:41)\n    at processImmediate (internal/timers.js:461:21)","message":"Zip version 78.8 is not supported","name":"Error","image":"sg-armv7-rpi-buster-2021-320.zip"}
gui.js:90033 Terminating IPC server
```

**Comment by Nmatt1** [comment]
From what the error is saying, it looks like Etcher wants a ISO / IMG file and not a zip file. When telling it to close from URL it is 100% expecting to get an ISO and you are not providing it. That is why when you download the zip and then feed it the ISO / IMG it works.

TL;DR feed it an ISO / IMG only when doing clone via url
